docs/src/handle.rst

```diff
@@ -17,6 +17,33 @@ Data types
 
     The base libuv handle type.
 
+.. c:type:: uv_handle_type 
+
+    The kind of the libuv handle.
+
+    ::
+        typedef enum {
```

Comment:
you need to leave a newline here, otherwise sphinx doesn't correctly format the code, IIRC